modules/mobile/app/workers/mobile/v0/vet360_linking_job.rb

```diff
@@ -25,10 +25,23 @@ class Vet360LinkingJob
       class MissingUserError < StandardError; end
 
       def perform(uuid)
-        user = IAMUser.find(uuid) || User.find(uuid)
+        user = IAMUser.find(uuid) || ::User.find(uuid)
         raise MissingUserError, uuid unless user
 
+        mobile_user = Mobile::User.find_or_create_by(icn: user.icn)
+
+        if mobile_user.vet360_linked
+          Rails.logger.info('This User previously had a vet360 ID linked',
+                            { icn: mobile_user.icn, attempts: mobile_user.vet360_link_attempts })
+          mobile_user.vet360_linked = false
+        end
+
+        mobile_user.increment_vet360_link_attempts
```

Comment:
I thought after would be good so that the log says "This was previously successful as of attempt number #" then increment, but ya the controller will log the attempt that was successful so I'm good with moving the increment